scripts/enabled_test_modules.py

```diff
@@ -198,7 +198,7 @@
     ],
     'from_db_value': [
         '"Cash" has no attribute',
-        'Argument 1 to "__str__" of "Decimal"',
+        '"__str__" of "Decimal"',
```

Comment:
Submitted PR to Django to fix their broken test code: https://github.com/django/django/pull/12726